test/smoketest_artman.py

```diff
@@ -101,10 +100,15 @@ def parse_args(*args):
              'repo in order for smoketest to run properly. If not specified, '
              'all APIs will be tested. APIs in the blacklist will not be '
              'tested.')
+    parser.add_argument(
+        '--input-dir',
+        default='/googleapis',
```

Comment:
This smoketests is supposed to be running inside artman docker container in most cases, where /googleapis is where the input will locate by default.